tool/site/client-react/src/components/prescription-summary/practitionerRoleSummaryList.tsx

```diff
@@ -58,24 +84,23 @@ interface SummaryOrganization {
 
 const PractitionerRoleSummaryList: React.FC<SummaryPractitionerRole> = ({
   name,
+  professionalCodes,
   telecom,
   organization,
   parentOrganization
 }) => {
-  const addressLineFragments = organization.addressLines.map((addressLine, index) => (
-    <Fragment key={index}>
-      {index > 0 && <br/>}
-      {addressLine}
-    </Fragment>
-  ))
+  const addressLineFragments = newLineFormatter(organization.addressLines)
+  const professionalCodeFragments = newLineFormatter(professionalCodes)
   return (
-    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-    // @ts-ignore
     <SummaryList>
       <SummaryList.Row>
         <SummaryList.Key>Name</SummaryList.Key>
         <SummaryList.Value>{name}</SummaryList.Value>
       </SummaryList.Row>
+      <SummaryList.Row>
+        <SummaryList.Key>Professional Codes</SummaryList.Key>
```

Comment:
Seems a bit odd to have:
**Professional Codes:** GMP Number - G123456

Rather than:
**GMP Number:** G123456

We could have `professionalCodes` be a map of id type to id value (or an array of pairs), and then make this a repeating row.